en: add tune-configuration.md and update dm-upgrade.md (#130)
pingcap/docs-dm: en/tune-configuration.md

### `chunk-filesize`

During full backup, DM splits the data of each table into multiple chunks according to the value of the `chunk-filesize` option. Each chunk is saved in a file with a size of about `chunk-filesize`. In this way, data is split into multiple files and you can use the parallel processing of the DM Load unit to improve the import speed. The default value of this option is 64 (in MB). Normally, you do not need to set this option. If you set it, adjust the value of this option according to the size of the full data.

> **Note:**
>
> -  You cannot update the value of `mydumpers` after the replication task is created. Be sure about the value of each option before creating the task. If you need to update the value, stop the task using dmctl, update the configuration file, and re-create the task.
> - `mydumpers`.`threads` can be replaced with the `mydumper-thread` configuration item for simplicity.
> - If `rows` is set，DM ignores the value of `chunk-filesize`.

## Full data import

`loaders` is the configuration item related to full data import. This section describes how to configure performance-related options.

### `pool-size`

> The `pool-size` option determines the number of threads in the DM Load phase. The default value is 16. Normally, you do not need to set this option. If you set it, adjust the value of this option according to the size of the full data and the performance of the database.

Reviewer: ```suggestion
The `pool-size` option determines the number of threads in the DM Load unit. The default value is 16. Normally, you do not need to set this option. If you set it, adjust the value of this option according to the size of the full data and the performance of the database.
```

> **Note:**
>
> -  You cannot update the value of `loaders` after the replication task is created. Be sure about the value of each option before creating the task. If you need to update the value, stop the task using dmctl, update the configuration file, and re-create the task.
> - `loaders`.`pool-size` can be replaced with the `loader-thread` configuration item for simplicity.

## Incremental data replication

`syncers` is the configuration item related to incremental data replication. This section describes how to configure performance-related options.

### `worker-count`

> `worker-count` determines the number of threads for concurrent replication of DMLs in the DM Sync phase. The default value is 16. To speed up data replication, increase the value of this option appropriately.

Reviewer: ```suggestion
`worker-count` determines the number of threads for concurrent replication of DMLs in the DM Sync unit. The default value is 16. To speed up data replication, increase the value of this option appropriately.
```

### `batch`

> `batch` determines the number of DMLs included in each transaction when the data is replicated to the downstream database during the DM Sync phase. The default value is 100. Normally, you do not need to change the value of this option.

Reviewer: ```suggestion
`batch` determines the number of DMLs included in each transaction when the data is replicated to the downstream database during the DM Sync unit. The default value is 100. Normally, you do not need to change the value of this option.
```

---

> title: DM configuration optimization

Reviewer 2: ```suggestion
title: Optimize Configuration of DM
```

> summary: Introduce how to optimize the configuration to improve the performance of data synchronize

Reviewer 2: ```suggestion
summary: Learn how to optimize the configuration of DM to improve the performance of data replication
```

Reviewer 2: please change `synchronize` to replicate, `synchronization` to `replication`~

category: reference
---

> # DM configuration optimization

Reviewer 2: ```suggestion
# Optimize Configuration of DM
```
